data/hai/hai reveal 4 middle.txt

```diff
@@ -1413,13 +1413,122 @@ mission "Hai Reveal [A11] Sound the Alarm"
+			`	Not visibly disheartened by the situation Teeneep steps into the middle and begins. She makes her case thoroughly and passionately, but with a clearly well-practiced presentation structure, ensuring that the full background for the situation is laid out for those present. Some elders make snide comments at various points but the moderator shuts them down, and those most skeptical settle for being merely disengaged. When she presents the Unfettered's position as an offer of assistance there is a smattering of largely disbelieving sounds, though when she speaks of their willingness to force the matter the tone of the conversation gets remarkably more tense. "If you don't act, they will. Whether that's with you, or through you," she finishes.`
+			`	One of the more stubborn Elders stands up, heralded by the moderator. "The Unfettered have made serious attempts to push through at Wah Ki before, to no avail. More recently, they've been more trouble again, and again we have organized more than enough security to keep their disruption contained, even with the current human crisis. This threat is ridiculous, and the Unfettered would not dare, not while they are so dependent on our food and medical supplies these past years. They would most certainly sacrifice many thousands of lives in the pursuit of misguided justice for a mere handful."`
+			`	"You are blind. Blind and old and foolish," snaps Teeneep, but another elder has leaped to their feet and the moderator cuts her off with a rebuke that you don't need translated to know what it means.`
+			`	After being properly heralded, the new elder addresses you. "I understand Teeneep has enlisted you as her gopher. You have my sympathy. I have no doubt that your continued involvement means that you accept her conspiracy theories that many of the pirate activities, specifically relating to kidnapping and people smuggling, are a coordinated effort. So I will not waste time retreading your position on points she has just laid out. I must ask you of this recent interaction though. Tell me, how would you characterize Teeneep's meeting with the Unfettered Choot'k: constructive, aggressive, or something else?"`
+			choice
+				`	"It was a constructive discussion."`
+					goto constructive
+				`	"I witnessed the test of two wills."`
+					goto willed
+				`	"The meeting was poignant."`
+					goto poignant
+				`	"Very aggressive and confronting."`
+			action
+				set haiignorance
+			`	You hear many sighs, and several people shrug, but nobody seems surprised, just disappointed. "They grow ever more brazen in their desire for change," says the Elder. "Thank you Captain <last>."`
+				goto next
+			label willed
+			action
+				set haiignorance
+			`	Many elders chuckle around the room. The Elder says, "Ah, our pair of old rivals, divided by Cloudfire. The embers are not yet quenched then." At this, Teeneep rolls her eyes and the Elder departs.`
```

Comment:
```suggestion
			`	Many elders chuckle around the room. The Elder says, "Ah, our pair of old rivals, divided by Cloudfire. The embers have not yet quenched then." At this, Teeneep rolls her eyes and the Elder departs.`
```
Not sure about this, if it's some quote or something.

Reply:
Yeah, I would like to know why, though.

Reply:
Embers do not quench themselves, it's a thing done to them. Therefore it cannot be "have" because that would be for processes that occur without interference.